build_projects/shared-build-targets-utils/Utils/Monikers.cs

```diff
@@ -15,9 +16,33 @@ public class Monikers
 
         public static string GetProductMoniker(BuildTargetContext c, string artifactPrefix, string version)
         {
-            string osname = GetOSShortName();
-            var arch = CurrentArchitecture.Current.ToString();
-            return $"{artifactPrefix}-{osname}-{arch}.{version}";
+            string rid = RuntimeEnvironment.GetRuntimeIdentifier();
+
+            if (rid == "ubuntu.16.04-x64" || rid == "fedora.23-x64" || rid == "opensuse.13.2-x64")
```

Comment:
consider moving these literals to static readonly values on a RID class instead of having future modifiers copy/paste these throughout the cs files